src/devices/cpu/powerpc/ppc_dasm.cpp

```diff
@@ -1169,18 +1169,19 @@ offs_t ppc_dasm_one(std::ostream &stream, uint32_t pc, uint32_t op)
 	return 4 | flags;
 }
 
-offs_t ppc_dasm_one(char *buffer, uint32_t pc, uint32_t op)
+
+offs_t ppc_dasm_one(std::string &string, uint32_t pc, uint32_t op)
```

Comment:
Can you eliminate this extra trampoline altogether?  It doesn't seem to serve much of a purpose.